mappings/net/minecraft/world/entity/ClientEntityManager.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_5582 net/minecraft/client/world/ClientEntityManager
+CLASS net/minecraft/class_5582 net/minecraft/world/entity/ClientEntityManager
```

Comment:
This class is still called `ClientEntityManager` even though it's been moved out of the client package. Is this correct?